Demo/DemoUITests/CardUITests.swift

```diff
@@ -0,0 +1,31 @@
+import XCTest
+
+class CardUITests: XCTestCase {
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        continueAfterFailure = false
+    }
+
+    func testApproveOrder() throws {
+        let app = launchApp(withArgs: ["-EnvironmentSandbox", "-DemoTypeCard"])
+
+        app.button(named: "Create Order").tap()
+
+        _ = app.staticText(containing: "Order ID:").waitForExistence(timeout: 10.0)
+
+        app.textField(named: "Card Number").tap()
+        app.typeText("4111111111111111")
+
+        app.textField(named: "Expiration").tap()
+        app.typeText("0223")
+
+        app.textField(named: "CVV").tap()
+        app.typeText("123\n")
+
+        app.button(named: "Capture Order").tap()
+
+        let elementExists = app.staticText(containing: "status: APPROVED").waitForExistence(timeout: 20.0)
```

Comment:
We should maybe consider standardizing the timeouts when we fully implement the tests

Reply:
Good call yeah. I made a helper that applies a default `20.0` second value.